MultiVI_tutorial.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>this would be good for a warning box, if you donwload one of the tutorials with a warniung box (the intro one I think), then double click the cell and it gives you the HTML you need</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>